docs/siteconfig.rst

```diff
@@ -242,3 +242,7 @@ Here is a sample configuration with many of the options set and documented::
     pelagos:
       endpoint: http://head.ses.suse.de:5000/
       machine_types: ['type1', 'type2', 'type3']
+
+    # Do not allow user to schedule that many jobs by default.
```

Comment:
Mentioning the user could be interpreted as some sort of per-user quota or limit.  Stress that it is per-run:
```suggestion
    # Do not allow more than that many jobs in a single run by default.
```